src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/LrdCourtVenueController.java

```diff
@@ -94,8 +94,10 @@
         @RequestParam(value = "region_id", required = false) @NotNull Integer regionId,
         @RequestParam(value = "cluster_id", required = false) @NotNull Integer clusterId,
         @RequestParam(value = "court_venue_name", required = false) @NotNull String courtVenueName) {
+        log.info("Inside retrieveCourtVenues");
```

Comment:
Is it worthwhile for us to add the logging component name (for all these loggers)?

Reply:
Added Logging Component name.